.gitlab-ci.yml

```diff
@@ -181,7 +179,7 @@ fedora:
 cuda10-maxset:
   <<: *global_job_definition
   stage: build
-  image: gitlab.icp.uni-stuttgart.de:4567/espressomd/docker/cuda:10.1
+  image: gitlab.icp.uni-stuttgart.de:4567/espressomd/docker/ubuntu-python3:cuda-10.1
```

Comment:
what naming convention are we following for the container images?

Reply:
We merged the cuda and scafacos dockerfiles. Because scafacos requires a build script, and docker can only access files in the current folder, we can either duplicate the scafacos build file in the cuda folder, or move the cuda dockerfile in the ubuntu folder. I opted for the latter. Once this PR is merged, I'll move the cuda9 dockerfile in the ubuntu folder, delete the old cuda:10 and cuda:tutorial dockerfiles, and the cuda folder.